packages/sdk-ui/src/components/network-selector/network-selector.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import { View } from 'react-native';
+import { ScrollView } from 'react-native-gesture-handler';
+import { AvatarVariant } from '../../design-system/components/Avatars/Avatar';
+import Cell, { CellVariant } from '../../design-system/components/Cells/Cell';
+
+// Internal dependencies
+import { useSDK } from '@metamask/sdk-react';
+import { Text } from 'react-native-paper';
+import styles from './NetworkSelector.styles';
+
+const t = require('node_modules/cryptocurrency-icons/svg/color/eth.svg');
+
+export interface NetworkSelectorProps {
+  showTestNetworks: boolean;
+}
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const NetworkSelector = ({ showTestNetworks }: NetworkSelectorProps) => {
+  const { chainId } = useSDK();
+
+  const onNetworkChange = (type: string) => {
+    console.log(`onNetworkChange: ${type}`);
+  };
+
+  const renderMainnet = () => {
+    const mainnetName = 'Ethereum';
+    return (
+      <Cell
+        variant={CellVariant.Select}
+        title={mainnetName}
+        avatarProps={{
+          variant: AvatarVariant.Network,
+          name: mainnetName,
+          imageSource: t,
+        }}
+        isSelected={chainId === '0x1'}
+        onPress={() => onNetworkChange(mainnetName)}
+        style={styles.networkCell}
+      />
+    );
+  };
+
+  // const renderLineaMainnet = () => {
```

Comment:
why all this commented functions and then the comments in the rendering? Is it not ready?